design/comments.go

```diff
@@ -60,16 +60,10 @@ var createCommentAttributes = a.Type("CreateCommentAttributes", func() {
 })
 
 var commentRelationships = a.Type("CommentRelations", func() {
-	a.Attribute("created-by", commentCreatedBy, "This defines the created by relation")
+	a.Attribute("creator", relationGeneric, "This defines the creator of the comment")
```

Comment:
> Although the relationGeneric can sure represent this structure it will make the swagger documentation less readable because you will not know what to expect.

In workitem design, creator uses `relationGeneric`, so shouldnt it also have something like `workitemCreator` structure similar to `commentCreator` to make the swagger documentation more readable?